heartbeat/galera

```diff
@@ -488,6 +501,65 @@ detect_first_master()
     set_bootstrap_node $best_node
 }
 
+galera_status()
+{
+    local loglevel=$1
+    local rc
+
+    if [ -e $OCF_RESKEY_pid ]; then
+        mysql_common_status $loglevel
+        rc=$?
+    else
+        # if pidfile is not created, the server may
+        # still be starting up, e.g. running SST
+        ps auxww | grep -v -e "${OCF_RESKEY_binary}" -e grep | grep -qe "--pid-file=$OCF_RESKEY_pid"
```

Comment:
so, we've given a solution to people when their galera was started by systemd, to copy /var/run/mariadb.pid to /var/run/mysqld.pid,which wouldn't be caught by this new test.  what do we lose if we just grep for "mysqld" directly, rather than excluding "mysqld_safe" etc. ?